### PR TITLE
first commit home work 11

### PR DESCRIPTION
**Pull request by Oleksii1985**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,8 +5,8 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <script src="lesson_4/customerForm.js" defer></script>
-  <title>home work 8</title>
+  <script src="lesson_5/difficultCycles.js"></script>
```
Comment (web-scavenger) [inline index.html:8]: This js insertion will block the render((((

File: lesson_5/difficultCycles.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
+// Вивести числа від 20 до 30 через пропуск, використовуючи крок 0,5 (20 20,5 21 21,5….).
+const result = [];
+
+for (let i = 20; i < 31;) {
+  result.push(i);
+  i = i + 0.5;
+}
+console.log(result);
```
Comment (web-scavenger) [inline lesson_5/difficultCycles.js:8]: This is incorrect solution
the result have to be 
`'20 20,5 21 21,5....'`
your result 
`[20, 20,5, 21 ...]`

File: lesson_5/difficultCycles.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+// Вивести числа від 20 до 30 через пропуск, використовуючи крок 0,5 (20 20,5 21 21,5….).
+const result = [];
+
+for (let i = 20; i < 31;) {
+  result.push(i);
+  i = i + 0.5;
```
Comment (web-scavenger) [inline lesson_5/difficultCycles.js:6]: why did you move this inside the body?

File: lesson_5/difficultCycles.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
+// Вивести числа від 20 до 30 через пропуск, використовуючи крок 0,5 (20 20,5 21 21,5….).
+const result = [];
+
+for (let i = 20; i < 31;) {
```
Comment (web-scavenger) [inline lesson_5/difficultCycles.js:4]:
```suggestion
for (let i = 20; i < 31; i+=0.5) {
```

File: lesson_5/difficultCycles.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+
+// Один долар коштує 27 гривень. Вивести дані з розрахунком вартості 10, 20, 30... 100 доларів.
+const hryvnya = 27;
+const resultDollar = [];
+
+for (let i = 10; i < 101;) {
+  let result = hryvnya * i;
+  resultDollar.push(result);
+  i = i + 10;
```
Comment (web-scavenger) [inline lesson_5/difficultCycles.js:18]: same
https://github.com/Oleksii1985/Voitiuk_front_end_26102022/pull/8/files#r1014878735

File: lesson_5/difficultCycles.js [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  i = i + 0.5;
+}
+console.log(result);
+
+
+// Один долар коштує 27 гривень. Вивести дані з розрахунком вартості 10, 20, 30... 100 доларів.
+const hryvnya = 27;
+const resultDollar = [];
+
+for (let i = 10; i < 101;) {
```
Comment (web-scavenger) [inline lesson_5/difficultCycles.js:15]: it wil be better i <= 100

File: lesson_5/difficultCycles.js [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+
+// Дане ціле число. Вивести всі цілі числа від 1 до 100, квадрат яких не перевищує числа N.
+const number = 250;
+const resultNumber = [];
+
+
+for (let i = 1; i < 100; i++) {
+  let res = i ** 2;
+  let num = i;
```
Comment (web-scavenger) [inline lesson_5/difficultCycles.js:30]: do you really need this variable? `num`

File: lesson_5/difficultCycles.js [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+  resultNumber.push(num);
+}
+console.log(resultNumber);
+
+
+// Дане ціле число. З'ясувати, чи є воно простим (простим називається число, більше 1, які не мають інших дільників крім 1 і себе).
+
+function isPrime(n) {
+  if (n < 2) {
+    return 'Число має бути більше 2';
```
Comment (web-scavenger) [inline lesson_5/difficultCycles.js:43]: `більше 1` по умові

File: lesson_5/difficultCycles.js [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+  return 'Просте число';
+}
+
+console.log(97, isPrime(97));
+
+
+// Дане деяке число. Визначити, чи можна одержати це число шляхом зведення числа 3 у деякий ступінь. (Наприклад, числа 9, 81 можна отримати, а 13 - не можна).
+
+let exponentialNumber = function (n) {
```
Comment (web-scavenger) [inline lesson_5/difficultCycles.js:65]: I guess that this solution is quite complicated(
This task will work only for 3

**Review by web-scavenger — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Oleksii1985) [reply index.html:8]: I should put defer?

File: lesson_5/difficultCycles.js (hunk 2 above)
Comment (Oleksii1985) [reply lesson_5/difficultCycles.js:8]: ok

File: lesson_5/difficultCycles.js (hunk 3 above)
Comment (Oleksii1985) [reply lesson_5/difficultCycles.js:6]: ok

File: lesson_5/difficultCycles.js (hunk 4 above)
Comment (Oleksii1985) [reply lesson_5/difficultCycles.js:4]: ok

File: lesson_5/difficultCycles.js (hunk 5 above)
Comment (Oleksii1985) [reply lesson_5/difficultCycles.js:18]: ok

File: lesson_5/difficultCycles.js (hunk 6 above)
Comment (Oleksii1985) [reply lesson_5/difficultCycles.js:15]: ok

File: lesson_5/difficultCycles.js (hunk 7 above)
Comment (Oleksii1985) [reply lesson_5/difficultCycles.js:30]: I was tired maybe)
